index.html

```diff
@@ -97,20 +97,34 @@ <h2>Stages</h2>
     <td>Experimental
   </tr>
   <tr>
-    <td>3
-    <td>Indicate that further refinement will require feedback from implementations and users
+    <td>2.7
+    <td>Gain initial experience with the feature through testing and prototyping
     <td>
       <ul>
         <li>Above
         <li>Complete spec text
         <li>Designated reviewers have signed off on the current spec text
-        <li>All ECMAScript editors have signed off on the current spec text
+        <li>ECMAScript editors have signed off on the current spec text
+      </ul>
+    </td>
+    <td>The solution is complete and no further work is possible without feedback from tests, implementations, or usage. No changes to the proposal will be requested aside from those elicited through testing or implementation experience.
```

Comment:
```suggestion
    <td>The solution is complete and no further work is possible without feedback from tests, implementations, or usage. No changes to the proposal will be requested aside from those elicited through testing, implementation, or usage experience.
```
surely? given that "usage" is mentioned in the previous sentence and multiple times below.

Reply:
nit: put implementation before testing